docs/sql-ref-ansi-compliance.md

```diff
@@ -648,6 +648,7 @@ Below is a list of all the keywords in Spark SQL.
 |PRECEDING|non-reserved|non-reserved|non-reserved|
 |PRIMARY|reserved|non-reserved|reserved|
 |PRINCIPALS|non-reserved|non-reserved|non-reserved|
+|PROCEDURES|reserved|non-reserved|non-reserved|
```

Comment:
Do we have examples of other databases where `PROCEDURES` is a reserved keyword? What does the SQL standard say?

Reply:
Changed to non-reserved